provider/equinix/environ_network.go

```diff
@@ -0,0 +1,200 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package equinix
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/juju/errors"
+	"github.com/juju/juju/core/instance"
+	"github.com/juju/juju/core/network"
+	corenetwork "github.com/juju/juju/core/network"
+	"github.com/juju/juju/environs"
+	"github.com/juju/juju/environs/context"
+	"github.com/juju/juju/environs/instances"
+	"github.com/juju/names/v4"
+)
+
+var _ environs.Networking = (*environ)(nil)
```

Comment:
I thought that we agreed to split the PR into two parts and land the `Networking` interface implementation as a separate PR.

Reply:
Yes we did, but when I was submitting the PR I realized that there are only two functions implemented in this file. Having a separate PR for so little felt like a overkill. 

If you want I can take this out of this PR

Reply:
Yes, please remove that to reduce the PR size and make it easier to review for others. It is not required to test the basic provider behavior (bootstrap-deploy-teardown); the Networking interface enables support for more advanced features such as spaces and can be tested separately.

Reply:
done